server/interpreter.js

```diff
@@ -2306,9 +2310,8 @@ Interpreter.prototype['stepCallExpression'] = function() {
   }
   if (!state.doneExec_) {
     state.doneExec_ = true;
-    if (state.func_ === null || state.func_ === undefined) {
-      this.throwException(this.TYPE_ERROR,
-          (typeof state.func_) + ' is not a function');
+    if (!state.func_ || !this.isa(state.func_, this.FUNCTION)) {
```

Comment:
This check is wrong.  Probably best just to just check we have an object here, and throw 'not a function' below if you can't figure out how to call it.